backend/api/v1/schema_design_transformer_tidb.go

```diff
@@ -259,7 +259,7 @@ func tidbRestoreFieldType(fieldType *tidbtypes.FieldType) (string, error) {
 		return fieldType.CompactStr(), nil
 	}
 	var buffer strings.Builder
-	flag := tidbformat.RestoreKeyWordLowercase
+	flag := tidbformat.RestoreKeyWordLowercase | tidbformat.RestoreStringSingleQuotes
```

Comment:
How about based on the DEFAULT flag?

Reply:
It's ok, but the generated schema type will be uppercase. the baseline schema and database metadata are lowercase at most time.